FirebaseDatabase/Tests/Unit/Swift/DatabaseAPITests.swift

```diff
@@ -112,7 +112,7 @@ final class DatabaseAPITests {
       let /* dataSnapshot */ _: DataSnapshot = dataSnapshot
     }
 
-    #if compiler(>=5.5) && canImport(_Concurrency)
+    #if compiler(>=5.5.2) && canImport(_Concurrency)
       if #available(iOS 15.0, macOS 12.0, macCatalyst 15.0, tvOS 15.0, *) {
```

Comment:
We're inconsistent with watchOS versions.  The Package.swift requires 7.0. Most, but not all of the podspecs are 6. Perhaps, we don't need to specify it anywhere?  Or we should be consistent with 6 or 7?

Reply:
I'm fine to be consistent with 7. By not mentioning it, it is included for all version (`*`) which is incorrect but I'm guessing we just didn't see it since we didn't run the tests on watchOS where this didn't work. I'll update them.

Reply:
Done in the latest commit.